parsers/gob.go

```diff
@@ -0,0 +1,43 @@
+package parsers
+
+import (
+	"encoding/gob"
+	"fmt"
+	"os"
+
+	"github.com/ablqk/littre-bot/src/dictionary"
+)
+
+var (
+	gobFile = "bin/dict.gob"
+)
+
+func ParseGob() ([]dictionary.Entry, error) {
```

Comment:
UT it (maybe a single test, covering save + parse, to avoid storing binary files on github)

Reply:
More importantly, you want to test that these two work together. If you change the binary format, you don't want to update references.

Reply:
but a .gob file could have been written by a previous run. If we change the parsing after the writing of the file, we'll face sever issues